README.Rmd

```diff
@@ -14,69 +14,98 @@ knitr::opts_chunk$set(
 ```
 
 # rlena
-[![Project Status: Concept - Minimal or no implementation has been done yet.](http://www.repostatus.org/badges/0.1.0/concept.svg)](http://www.repostatus.org/#concept)
+[![Project Status: WIP – Initial development is in progress, but there has not yet been a stable, usable release suitable for the public.](http://www.repostatus.org/badges/latest/wip.svg)](http://www.repostatus.org/#wip)
 
```

Comment:
When you get travis running, use `devtools::use_travis()` to get the code to paste in for the travis badge

Reply:
I added the badges that point to the master branch of my fork for now. Of course it makes more sense to activate Travis for the upstream repo, i.e. `HomeBankCode/rlena`. I suppose you can do that? 
I've also added coveralls support. It also has to be activated for the repo.
Travis and Coveralls work for he fork. The badges show "status unknown" though, because they point to the master whereas the cleanup branch is the one being checked.

Reply:
I put in a request to have higher privileges in the HomeBankCode, so I can turn on travis for this repository.